Add devd config for using acpi_video as a source

cperciva opened the pull request:
On my Dell 7390 laptop, the brightness hotkey (Fn+Up, Fn+Down) trigger
ACPI events which result in acpi_video adjusting its "brightness" value
-- but do not cause the brightness to actually change.  This devd
config takes notifications from acpi_video and uses intel_backlight to
perform the actual brightness adjustment.

cperciva committed:
Add devd config for using acpi_video as a source

On my Dell 7390 laptop, the brightness hotkey (Fn+Up, Fn+Down) trigger
ACPI events which result in acpi_video adjusting its "brightness" value
-- but do not cause the brightness to actually change.  This devd
config takes notifications from acpi_video and uses intel_backlight to
perform the actual brightness adjustment.

0mp commented on acpi-video-intel-backlight.conf:
```diff
@@ -0,0 +1,6 @@
+        match "system" "ACPI";
+        match "subsystem" "Video";
+        match "type" "brightness";
+        action "/usr/local/bin/intel_backlight $notify";
```
From the ports perspective, it would be nice to be able to replace /usr/local with PREFIX during building.

cperciva replied:
I guess I could do that.  It makes the port slightly messier though; I was planning on adding this file to PORTEXAMPLES so that it would be installed along with the two shell scripts.

0mp replied:
Oh that's probably a fine approach as well.

cperciva replied:
On further consideration, I'd prefer to keep `/usr/local` hard-coded here and have the port replace it by PREFIX.  Can you pull this now?

0mp replied:
I cannot merge any changes here. I'm just a watcher ;)

zeising commented:
Is there any reason this can't be hooked up to the Makefile here, and at the same time we can take care of hardcoding /usr/local?

With regards to FreeBSD ports, portmgr usually frowns on hardcoding /usr/local instead of using ${PREFIX}, so I'd prefer if we did it right here, instead of adding a REINPLACE_CMD to the port.

cperciva commented:
> Is there any reason this can't be hooked up to the Makefile here, and at the same time we can take care of hardcoding /usr/local?

I figured that it made the most sense to have this installed the same way as the other two "read settings and invoke intel_backlight" scripts.

> With regards to FreeBSD ports, portmgr usually frowns on hardcoding /usr/local instead of using ${PREFIX}, so I'd prefer if we did it right here, instead of adding a REINPLACE_CMD to the port.

I guess we could do it here.  Doing it in the port seemed easier.

cperciva committed:
Fix ${PREFIX} in acpi-video-intel-backlight.conf

Handle replacing /usr/local by ${PREFIX} in acpi-video-intel-backlight.conf
here rather than in ports/graphics/intel-backlight/Makefile.

cperciva commented:
Ok, can you merge this now?
